pkg/server/router/presentation.go

```diff
@@ -444,3 +434,143 @@ func (pr PresentationRouter) ReviewSubmission(c *gin.Context) error {
 	}
 	return framework.Respond(c, ReviewSubmissionResponse{Submission: submission}, http.StatusOK)
 }
+
+const DefaultExpirationDuration = 30 * time.Minute
+
+type CreateRequestRequest struct {
+	// Audience as defined in https://www.rfc-editor.org/rfc/rfc7519.html#section-4.1.3
+	// Optional
+	Audience []string `json:"audience"`
+
+	// Expiration as defined in https://www.rfc-editor.org/rfc/rfc7519.html#section-4.1.4
+	// Optional. When not specified, the request will be valid for 30 minutes.
+	Expiration string `json:"expiration"`
```

Comment:
hmm...what do you think about turning this off by default? I feel like no expiry is a more common behavior

Reply:
I think it's less secure to have it off by default. I don't know what will be more common tbh, but as we have them, presentation requests are meant to be specific to an audience, and that means it's likely short lived.

Reply:
less secure how? `exp` is an optional claim so seems odd to make it required

Reply:
It reduces the time span during which an attacker can intercept the request and impersonates the receiver of the submission.

Reply:
Unless they can steal or break a key this isn't a realistic threat. What would most likely happen is the requester would have to create and manage a new PR which is more overhead

Reply:
Done